src/main/java/jalau/at18/katas/pokerhands/daniela/FourofaKind.java

```diff
@@ -1,28 +1,21 @@
 package jalau.at18.katas.pokerhands.daniela;
 
 public class FourofaKind extends CardsPattern {
-    private final int threecards = 4;
+    private final int four = 4;
     private final int othercard = 1;
-    private CardValue value1;
-    private CardValue value2;
 
-    public FourofaKind(CardValue value1, CardValue value2) {
-        this.value1 = value1;
-        this.value2 = value2;
+    public FourofaKind() {
     }
 
     @Override
     public boolean match(PokerHand handWith4SimilarCards) {
-        onePairs(handWith4SimilarCards);
-        return onePairs(handWith4SimilarCards);
+        fourKind(handWith4SimilarCards);
```

Comment:
This line can be removed

Reply:
already removed the line in the classes that was duplicated